scripts/copyright_update.py

```diff
@@ -70,7 +70,7 @@ def edit_files(files: List[str], copyright_line: str):
 
 def main():
     """Run the copyright updater"""
-    edited = list(edit_files(arguments(), r"2021-{}".format(date.today().year)))
+    edited = list(edit_files(arguments(), rf"2021-{date.today().year}"))  # noqa: DTZ011
```

Comment:
What is the r here for?

Reply:
The raw text encoding here (I'm guessing you knew that bit) is because edit files reads all files without encoding/decoding the strings so you need to pass a non escaped version of the string. 

Sorry I can't seem to find a good explanation link